### PR TITLE
Add few things to README.md

### PR DESCRIPTION
**Pull request by dd-create**
I add and change few line. Please check if it's okay or not.
@lin9703 @6loss0m @dlwlstks96

### CODE REVIEW

File: app/src/androidTest/java/com/euphonyforever/looking_for_job/ButtonTest.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            .perform(click())
+
+        intended(hasComponent())
+        intended(toPackage("com.android.phone"))
+    }
+
+    @Test
+    fun SendButtonClick2() {
+        val activityScenarioRule = activityScenarioRule<InputResumeInfo>()
+        // Unresolved reference: activityScenarioRule
```
Comment (yeonns) [inline app/src/androidTest/java/com/euphonyforever/looking_for_job/ButtonTest.kt:36]: If you resolved issue, I think it is better to remove this comment line :)

File: app/src/androidTest/java/com/euphonyforever/looking_for_job/ButtonPressTest.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            new IntentsTestRule<>(MainActivity.class);
+    // Cannot resolve symbol 'IntentsTestRule'
+
+    @Test
+    public void useAppContext() {
+        Intents.init();
+
+        onView(withId(R.id.btnSend)).perform(click());
+        intended(toPackege("com.euphonyforever.looking_for_job.InputResumeInfo"));
+        // Cannot resolve method 'intended' in 'ButtonPressTest'
```
Comment (yeonns) [inline app/src/androidTest/java/com/euphonyforever/looking_for_job/ButtonPressTest.java:49]: If you resolved issue, I think it is better to remove this comment line :)

**Review by yeonns — COMMENTED** [review]
There are some comment lines which is not necessary. I think it is better to remove them. 😄

**Comment by dlwlstks96** [comment]
The README is more detailed.
Great!

**Review by dlwlstks96 — APPROVED** [review]

**Review by lin9703 — APPROVED** [review]
It is better. :)

File: app/src/androidTest/java/com/euphonyforever/looking_for_job/ButtonPressTest.java (hunk 2 above)
Comment (dd-create) [reply app/src/androidTest/java/com/euphonyforever/looking_for_job/ButtonPressTest.java]: Thanks! Unfinished one commited together. I fixed it.

**Comment by dd-create** [comment]
> It is better. :)

Thanks for feedback. Can I merge this?

**Comment by lin9703** [comment]
> > It is better. :)
> 
> Thanks for feedback. Can I merge this?

Of course. It is okay to merge.

**Pull request merged**